.github/workflows/test.yml

```diff
@@ -43,6 +43,38 @@ jobs:
     - run: docker pull $DOCKER_IMAGE
     - run: docker run --rm -tv $(pwd):/src -e CI $DOCKER_IMAGE /src/${{ matrix.task}}/run.sh
 
+  lin-make:
+    strategy:
+        fail-fast: false
+        max-parallel: 3
+        matrix:
+          task: [
+            # vhpidirect/quickstart/random,
+            # vhpidirect/quickstart/math,
+            # vhpidirect/quickstart/customc,
+            # vhpidirect/quickstart/wrapping/basic,
+            # vhpidirect/quickstart/wrapping/time,
+            # vhpidirect/quickstart/linking/bind,
+            # vhpidirect/quickstart/package,
+            # vhpidirect/quickstart/sharedvar,
+            # vhpidirect/shared/shlib,
+            # vhpidirect/shared/dlopen,
+            # vhpidirect/shared/shghdl,
+            vhpidirect/arrays/intvector,
+            # vhpidirect/arrays/logicvector,
+            # vhpidirect/arrays/matrices,
+            # vhpidirect/arrays/matrices/framebuffer,
+            # vpi/quickstart,
+          ]
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
+      DOCKER_IMAGE: ghdl/ghdl:buster-llvm-7
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker pull $DOCKER_IMAGE
+    - run: docker run --rm -tv $(pwd):/src -e CI $DOCKER_IMAGE /src/${{ matrix.task}}/make run
```

Comment:
This could be fixed as `docker run --rm -tv $(pwd):/src -e CI -w /src/${{ matrix.task}} $DOCKER_IMAGE make run`.

However, I suggest to rebase on top of #25. There, you can leave the workflow untouched and just need to change the corresponding `check_call` in https://github.com/umarcor/ghdl-cosim/blob/pytest/test.py. For example:

```py
# from
check_call(self.shell + [str(self.vhpidirect / 'arrays' / 'intvector' / 'run.sh')], shell=True)

# to
check_call(self.shell + ['make', 'run'], shell=True, cwd=str(self.vhpidirect / 'arrays' / 'intvector'))
```

Reply:
By the number of things that broke within the workflow, I suppose I should wait until #25 is confirmed and merged? Or have I done something wrong?